gadgets/trace_tcpdrop/README.mdx

```diff
@@ -0,0 +1,211 @@
+    </TabItem>
+
+    <TabItem value="ig" label="ig">
+        In terminal 1, start the trace tcpdrop gadget:
+
+        ```bash
+        $ sudo -E ig run trace_tcpdrop:latest --containername test-trace-tcpdrop
+        RUNTIME.CONTAINERNAME      SRC                                DST                                MOUNT_NS_ID    COMM                      PID            TID STATE         TCPFLAGS      REASON        TIMESTAMP
+        ```
+
+        In terminal 2, start a container, configure the network emulator to drop 25% of the packets, and download a web page:
+
+        ```bash
+        $ docker run -ti --cap-add NET_ADMIN --name=test-trace-tcpdrop wbitt/network-multitool -- /bin/bash
+        # tc qdisc add dev eth0 root netem drop 25%
+        # wget 1.1.1.1
+        ```
+
+        The container needs NET_ADMIN capability to manage network interfaces
+
+        The results in terminal 1 will show that some packets are dropped by the network emulator qdisc:
+        ```
+        RUNTIME.CONTAINERNAME                                                       SRC                                                  DST                                                  MOUNT_NS_ID                              COMM                                                  PID                                     TID STATE                                   TCPFLAGS                                REASON                                  TIMESTAMP
+        test-trace-tcpdrop                                                          172.17.0.2:52508                                     1.1.1.1:80                                           4026533225                               wget                                               854278                                  854278 tcp_established                         ACK                                     SKB_DROP_REASON_QDISC_DROP              2024-08-02T12:35:20.188383289+02:00
+        test-trace-tcpdrop                                                          172.17.0.2:37838                                     1.1.1.1:443                                          4026533225                               wget                                               854278                                  854278 tcp_established                         ACK                                     SKB_DROP_REASON_QDISC_DROP              2024-08-02T12:35:20.597218523+02:00
+        test-trace-tcpdrop                                                          172.17.0.2:37838                                     1.1.1.1:443                                          4026533225                               wget                                               854278                                  854278 tcp_fin_wait1                           FIN|ACK                                 SKB_DROP_REASON_QDISC_DROP              2024-08-02T12:35:25.659670258+02:00
+        ```
+    </TabItem>
+</Tabs>
+
+### List of drop reasons
```

Comment:
I think it shouldn't be part of "guide" but more general documentation of the gadget.